doc/manual/manual.tex

```diff
@@ -6310,11 +6310,78 @@ \subsubsection{The van Keken thermochemical composition benchmark}
+
+\begin{figure}
+	\centering
+	\includegraphics[width=0.8\textwidth]{cookbooks/benchmarks/van-keken/smoothing-parameter.png}
+	\caption{\it Van Keken Benchmark using smoothed out interface: compositional field at time t=0 using smoothing parameter size: a) 0.0039, b) 0.0078, c) 0.0156, d) 0.0234, e) 0.0312, f) 0.0390, g) 0.0468, h) 0.0546, i) 0.0624.}
+	\label{fig:vk-5}
+\end{figure}
+
+The evolution in time of the resulting compositional fields (Fig.~\ref{fig:vk-6}) shows that the first peak converges as the smoothed interface decreases. There is a good agreement for the first peak for all smoothing parameters.  As the width of the discontinuity increases, the second peak rises both later and more slowly.
+
+\begin{figure}
+	\centering
+	\includegraphics[width=0.75\textwidth]{cookbooks/benchmarks/van-keken/smoothing-parameter-velocity.png}
+	\caption{\it Van Keken benchmark with smoothed initial conditions for the compositional field using refinement degree 7 for different smoothing parameters.}
+	\label{fig:vk-6}
+\end{figure}
+
+Now let us further add a two-layer viscosity model to the domain. Let's assume the viscosity value of the upper heavier layer is $\eta_{t}$ and the viscosity value of the lower lighter layer is $\eta_{b}$. Based on the initial constant viscosity value 1$\times10^{2}$ Pa~s, we set the viscosity proportion $\frac{\eta_{t}}{\eta_{b}}=0.1, 0.01$, meaning the viscosity of the upper, heavier layer is still 1$\times10^{2}$ Pa~s, but the viscosity of the lower, lighter layer is now either 10 or 1 Pa~s, respectively. The viscosity profiles of the discontinuous and smooth models are shown in Fig.~\ref{fig:vk-7}.
+
+\begin{figure}
+	\centering
+	\includegraphics[width=1.0\textwidth]{cookbooks/benchmarks/van-keken/contrast_viscosity.png}
+	\caption{\it Van Keken benchmark using layers of different viscosities. The left image is the discontinuous case, while right is the smooth.  Both are shown at t=0.}
+	\label{fig:vk-7}
+\end{figure}
+
+The end time, t=2000 s, for both benchmark cases, discontinuous and smooth, and both viscosity proportions, 0.1 and 0.01, are shown in Fig.~\ref{fig:vk-8}.  This was generated using the original input parameter file, running the cases in 8 global refinement degree, and also adding the two-layer viscosity model.
```

Comment:
"The end time ... are shown in Fig....". That's not actually what you show. You should the *state* at the end time.

"...in 8 global refinement degree" -> with 8 global refinement steps